src/types.js

```diff
@@ -11,44 +11,47 @@ export type TXOBJ = {
     prevScriptPubKey: string,
   }[],
   outs: { script: string, satoshis: number, isFromBackwardTransfer?: boolean, pubKeyHash?: string }[],
-  sc_params?: {
-    vsc_ccout: {
-      epoch_length: number,
-      satoshis: number,
-      address: string,
-      customData: string,
-      certVk: string,
-      constantData: string,
-      wCeasedVk: string,
-      vFieldElementCertificateFieldConfig: number[],
-      vBitVectorCertificateFieldConfig: number[][],
-      forwardTransferScFee: number,
-      mainchainBackwardTransferScFee: number,
-      mbtrRequestDataLength: number
-    }[],
-    vft_ccout: {
-      scid: string,
-      satoshis: number,
-      address: string,
-      mcReturnAddress: string
```

Comment:
@loucaponi @MarcoOl94 This field has been removed intentionally? It is part of the vft_ccout right? At least in bitcore-lib-zen we are adding it 🤔

Reply:
Yes this fhield should be kept.